test/src/main/java/org/apache/accumulo/test/ComprehensiveTableOperationsIT.java

```diff
@@ -0,0 +1,957 @@
+
+    Set<Text> rowsSeenBeforeNewOp = new HashSet<>();
+    try (var scanner = client.createScanner(AccumuloTable.FATE.tableName())) {
+      for (var entry : scanner) {
+        rowsSeenBeforeNewOp.add(entry.getKey().getRow());
+      }
+    }
+
+    // start a very slow compaction to create a FATE op that will linger in the FATE table until
+    // cancelled
+    client.tableOperations().compact(table, null, null, true, false);
+
+    Set<Text> rowsSeenAfterNewOp = new HashSet<>();
+    try (var scanner = client.createScanner(AccumuloTable.FATE.tableName())) {
+      for (var entry : scanner) {
+        rowsSeenAfterNewOp.add(entry.getKey().getRow());
+      }
+    }
+
+    var newOp = Sets.difference(rowsSeenAfterNewOp, rowsSeenBeforeNewOp);
+    assertEquals(1, newOp.size());
+    return newOp.stream().findFirst().orElseThrow();
+  }
+
+  private void attachSlowMajcIterator(String table) throws Exception {
+    if (!ops.listIterators(table).containsKey(SLOW_ITER_NAME)) {
+      IteratorSetting is = new IteratorSetting(1, SLOW_ITER_NAME, SlowIterator.class);
+      is.addOption("sleepTime", "60000");
+      ops.attachIterator(table, is, EnumSet.of(IteratorUtil.IteratorScope.majc));
+    }
+  }
+
+  private void removeSlowMajcIterator(String table) throws Exception {
+    ops.removeIterator(table, SLOW_ITER_NAME, EnumSet.of(IteratorUtil.IteratorScope.majc));
+  }
+
+  /**
+   * Creates some data in the SCAN_REF table. Will create a single row each call.
+   *
+   * @return the row created
+   */
+  private Text createScanRefTableRow() {
+    HostAndPort server = HostAndPort.fromParts("127.0.0.1", 1234);
+    UUID serverLockUUID = UUID.randomUUID();
+    Set<ScanServerRefTabletFile> scanRefs = Stream.of("F0000070.rf", "F0000071.rf")
+        .map(f -> "hdfs://localhost:8020/accumulo/tables/2a/default_tablet/" + f)
+        .map(f -> new ScanServerRefTabletFile(f, server.toString(), serverLockUUID))
+        .collect(Collectors.toSet());
+    getCluster().getServerContext().getAmple().scanServerRefs().put(scanRefs);
+    return new Text(getCluster().getServerContext().getAmple().scanServerRefs().list()
+        .filter(tf -> tf.getServerLockUUID().equals(serverLockUUID)).findFirst().orElseThrow()
+        .getServerLockUUID().toString());
+  }
+
+  /**
+   * Cleans up the data in the FATE table that was created by calls to
+   * {@link #createFateTableRow(String)}
+   */
+  private void cleanupFateTable() throws Exception {
+    ops.cancelCompaction(userTable);
+    // Wait for FATE table to be clear
+    Wait.waitFor(() -> {
+      try (var scanner = client.createScanner(AccumuloTable.FATE.tableName())) {
+        return !scanner.iterator().hasNext();
+      }
+    });
+  }
+
+  /**
+   * Cleans up the data in the SCAN_REF table that was created by calls to
+   * {@link #createScanRefTableRow()}
+   */
+  private void cleanupScanRefTable() throws Exception {
+    ops.deleteRows(AccumuloTable.SCAN_REF.tableName(), null, null);
```

Comment:
Same comment about deleteRows() here, but using it for now since it's helpful cleanup in this test

Reply:
Created https://github.com/apache/accumulo/pull/5491

Reply:
Replaced deleteRows here. deleteRows on system tables is no longer possible